docs/manual/manual.adoc

```diff
@@ -445,6 +445,41 @@ Ident   CCE-3967-7
 Result  pass
 ----
 
+The meaning of results is defined by https://csrc.nist.gov/CSRC/media/Publications/nistir/7275/rev-4/final/documents/nistir-7275r4_updated-march-2012_clean.pdf[XCCDF Specification].
+This table lists the possible results of a single rule:
+
+.XCCDF results
+|===
+|Result |Description
```

Comment:
Does it make sense to separate the description from the example into a new column?

Reply:
Yes, it can probably improve the orientation for the reader.